hls4ml/backends/vivado/passes/conv_same_pad.py

```diff
@@ -1,12 +1,12 @@
-from hls4ml.model.layers import Conv1D, Conv2D, SeparableConv1D, SeparableConv2D
+from hls4ml.model.layers import Conv1D, Conv2D, DepthwiseConv1D, DepthwiseConv2D, SeparableConv1D, SeparableConv2D
 from hls4ml.model.optimizer import OptimizerPass
 
 
 class InsertZeroPaddingBeforeConv1D(OptimizerPass):
     name = 'insert_zero_padding_before_conv1d'
 
     def match(self, node):
-        is_match = isinstance(node, (Conv1D, SeparableConv1D)) and (
+        is_match = isinstance(node, (Conv1D, DepthwiseConv1D, SeparableConv1D)) and (
```

Comment:
This is not needed. Depthwise inherits from regular conv. (The inheritance itself is slightly questionable, but we have it.)